test/unit/spec/MfaVerify_spec.js

```diff
@@ -2315,30 +2318,38 @@ function (Okta,
               });
             });
             itp('starts poll after a delay of 6000ms', function () {
-              return setupOktaPush().then(function (test) {
-                spyOn(LoginUtil, 'callAfterTimeout').and.callFake(function() {
-                  // reducing the timeout to 100 so that test is fast.
-                  return setTimeout(arguments[0], 100);
+              var callAfterTimeoutStub;
+
```

Comment:
nit: Remove the newline please

Reply:
actually it was added deliberately

Reply:
Is there a reason for it? This seems like a one-off from the layout of our other specs.